src/auth/mod.rs

```diff
@@ -279,13 +280,13 @@ mod test {
     use rocket::fairing::AdHoc;
     use rocket::http::Header;
     use rocket::local::Client;
-    use rocket_contrib::json::JsonValue;
+    use rocket_contrib::{json, json::JsonValue};
 
```

Comment:
man, that looks weird. I get that it's including the `json!` macro as well as `::json::JsonValue`, just... ugh.

Reply:
Right, and don't macros share the same namespace now? Probably, but maybe the identifier includes the !? Which you then omit in the import (more discussion here https://www.reddit.com/r/rust/comments/9dw9zt/do_macros_and_functions_live_in_the_same/ ). Definitely weird.